docs/concepts/gardenlet.md

```diff
@@ -221,6 +221,14 @@ Hence, if the `Shoot` is deleted, also the `BackupEntry` resource gets deleted.
 In this case, the controller deletes the `extensions.gardener.cloud/v1alpha1.BackupEntry` resource in the seed cluster and waits until the responsible extension controller has deleted it.
 Afterwards, the finalizer of the `core.gardener.cloud/v1beta1.BackupEntry` resource is released so that it finally disappears from the system.
 
+### `NetworkPolicy` Controller
+
+The `NetworkPolicy` controller reconciles `NetworkPolicy`s in shoot namespaces in order to ensure access to Kubernetes API server. 
+
+The controller resolves the IP address of Kubernetes service in `default` namespace and creates egress `NetworkPolicy`s for it.
+
+For more details about `NetworkPolicy`s in Gardener please see [this document](network_policies.md).
+
```

Comment:
I think this section should be further below (you sneaked it into the `BackupEntry` section).

Reply:
U-Boot 😄